src/test/scala/mesosphere/marathon/core/deployment/DeploymentPlanTest.scala

```diff
@@ -422,6 +422,36 @@ class DeploymentPlanTest extends UnitTest with GroupCreation {
       Then("The deployment is not valid")
       validate(plan2)(f.validator).isSuccess should be(false)
     }
+
+    "Deployment plan treats change with 0 instances as scale-only change" in {
+      Given("An application update with command and scale changes")
+      val mongoId = "/test/database/mongo".toPath
```

Comment:
`mongo`? rly? :D

Reply:
It's a kind of database you'd like to scale to 0 instances as soon as possible :)

Reply:
oh snap 😂